pkg/apis/config/defaults.go

```diff
@@ -160,6 +164,8 @@ func NewDefaultsConfigFromConfigMap(config *corev1.ConfigMap) (*Defaults, error)
 // Defaults includes the default values to be populated by the webhook.
 type Defaults struct {
 	RevisionTimeoutSeconds int64
+
+	RevisionIdleTimeoutSeconds int64
```

Comment:
This should get a comment (the one above it prolly too but not necessarily in this PR)

Reply:
Added some comments